__tests__/buildFiles.test.js

```diff
@@ -93,13 +93,13 @@ describe('buildFiles', () => {
 
   it('should throw if build path doesn\'t have a trailing slash', () => {
     expect(
-      buildFiles.bind(null, dictionary, platformWithBadBuildPath),
+      buildFiles.bind(null, dictionary, platformWithBadBuildPath)
```

Comment:
lets undo this format change for now, we can put in a separate PR if necessary

Reply:
The pre-commit hook (lint script) fails without this

Reply:
That is weird, it shouldn't. Can you be sure you are using our version of eslint and not a globally installed version? Double check you have `eslint` in node_modules/.bin and make sure you run `npm ci` to install exact dependency versions. You also might have an issue with your $PATH, you can run `npm run lint --verbose` to make sure local executables are before any global paths.